frame/uniques/src/lib.rs

```diff
@@ -329,6 +369,26 @@ pub mod pallet {
 		},
 		/// Ownership acceptance has changed for an account.
 		OwnershipAcceptanceChanged { who: T::AccountId, maybe_class: Option<T::ClassId> },
+		/// The price was set for the instance.
+		InstancePriceSet {
+			class: T::ClassId,
+			instance: T::InstanceId,
+			price: BalanceOrAssetOf<T, I>,
+			buyer: Option<T::AccountId>,
```

Comment:
```suggestion
			whitelisted_buyer: Option<T::AccountId>,
```
? since he did not buy it *yet*